hexrd/ui/workflow_selection_dialog.py

```diff
@@ -0,0 +1,36 @@
+from PySide2.QtCore import QObject
+
+from hexrd.ui.hexrd_config import HexrdConfig
+from hexrd.ui.ui_loader import UiLoader
+from hexrd.ui import constants
+
+
+class WorkflowSelectionDialog(QObject):
+
+    def __init__(self, parent=None):
+        loader = UiLoader()
+        self.ui = loader.load_file('workflow_selection_dialog.ui', parent)
+        self.init_gui()
+        self.update_gui_from_config()
+        self.setup_connections()
+
+    def init_gui(self):
+        for w in constants.WORKFLOWS:
+            self.ui.workflowComboBox.addItem(w)
+
+    def setup_connections(self):
+        self.ui.accepted.connect(self.accepted)
+        self.ui.rejected.connect(self.rejected)
+
+    def update_gui_from_config(self):
+        self.ui.workflowComboBox.setCurrentText(HexrdConfig().workflow)
+
+    def accepted(self):
+        HexrdConfig().workflow = self.ui.workflowComboBox.currentText()
```

Comment:
FYI, we came to the realization that on some operating systems (I don't know which ones), the text in the combo box can be altered by Qt (it adds things like ampersands). This would cause problems if anyone uses one of these operating systems.

I've personally resorted to always using item data since then for reliable string comparisons. When you add an item to the combo box via `addItem` (above), you can do `addItem(w, w)` to add it as item data, and then here, you can call `self.ui.workflowComboBox.currentData()` instead.

Reply:
(you might need to modify `update_gui_from_config()` as well, to find the item data that matches)

Reply:
I take these comments back. The issue we ran into earlier had to do with the label for a QCheckBox. It appears that labels can have things (like ampersands) automatically inserted into them, but I haven't found any evidence that QComboBox items can be tampered with.

We can leave this the way it is.